### PR TITLE
Configure eslint package and fix lint issues

### PR DESCRIPTION
**Pull request by JorikSchellekens**
TS is now linted

### CODE REVIEW

File: src/components/structures/VectorEmbeddedPage.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ export default class VectorEmbeddedPage extends EmbeddedPage {
     translate(s) {
         s = sanitizeHtml(_t(s));
         // ugly fix for https://github.com/vector-im/riot-web/issues/4243
+        // eslint-disable-next-line max-len
         s = s.replace(/\[matrix\]/, '<a href="https://matrix.org" target="_blank" rel="noreferrer noopener"><img width="79" height="34" alt="[matrix]" style="padding-left: 1px;vertical-align: middle" src="welcome/images/matrix.svg"/></a>');
```
Comment (turt2live) [inline src/components/structures/VectorEmbeddedPage.js:33]: why does this even exist 😭 

(not an issue for this PR, just expressing my sadness for such a regex)

File: src/vector/jitsi/index.ts [hunk 2]
```diff
@@ -23,7 +23,7 @@ import { Capability, WidgetApi } from "matrix-react-sdk/src/widgets/WidgetApi";
 // Dev note: we use raw JS without many dependencies to reduce bundle size.
 // We do not need all of React to render a Jitsi conference.
 
-declare var JitsiMeetExternalAPI: any;
+declare let JitsiMeetExternalAPI: any;
```
Comment (turt2live) [inline src/vector/jitsi/index.ts:26]: I'm somewhat sure we used `var` for a reason, but I guess we'll find out :D

**Comment by turt2live** [comment]
merging to try and unbreak develop

File: src/components/structures/VectorEmbeddedPage.js (hunk 1 above)
Comment (JorikSchellekens) [reply src/components/structures/VectorEmbeddedPage.js:33]: I have no idea

**Pull request merged**